### ISSUE TITLE
All pins are on one horizontal row when exporting PCB

### ISSUE THREAD

**Issue by RoelVB** [mcauser/Fritzing-Part-WeMos-D1-Mini#2]
When exporting the PCB all pin get on one horizontal row.

For this PCB:
![untitled sketch_pcb](https://cloud.githubusercontent.com/assets/10999128/17276216/80953fe4-5722-11e6-8403-0ae84a534378.png)

The export looks like this:
![untitled sketch_etch_copper_bottom](https://cloud.githubusercontent.com/assets/10999128/17276223/aa32aeea-5722-11e6-8bb2-61a68fb66c29.png)

**Comment by mcauser** [comment]
You're right. My SVG uses transform rotate, but it seems fritzing doesn't support it. I'll upload a fix soon.

**Comment by RoelVB** [comment]
I would fix is myself if I knew how.

**Comment by zen** [comment]
This is serious problem, any plans to fix it?
See:
http://forum.fritzing.org/t/wemos-d1-mini-pcb-protective-areas-rotated/2478

**Comment by BorisBrock** [comment]
I just ran into the same problem. A fix would be really great :-)

**Comment by mcauser** [comment]
@RoelVB fixed in https://github.com/mcauser/Fritzing-Part-WeMos-D1-Mini/commit/1b2e1d549bce96002bd27619f98f29bf2191672e

I'll make a PR on the fritzing-parts repo to cleanup the mess.
My original 4 parts were "fixed" by merging into 2 parts.
https://github.com/fritzing/fritzing-parts/commit/11ac5e9234c6bbef11e4330b5f7625d6aaed37c3
They got it wrong and put female headers on my male headers part.
I'll obsolete the older versions and call this one v3.

I came across this thread: http://forum.fritzing.org/t/wemos-d1-mini-pcb-protective-areas-rotated/2478
Hopefully this commit addresses all of the issues.

One part I am a little uncertain about is the drill sizes.
I used one of the Sparkfun boards as a template when creating my pcb svgs.
The 2x8 headers are 2.54mm spaced.
Is there another part with the correct size drill holes and solder mask that I can use for reference?

**Comment by RoelVB** [comment]
Sorry for the delay. Just made a PCB using your part. Everything looks fine.
